tests/kernel/fatal/exception/src/main.c

```diff
@@ -90,6 +90,35 @@ void entry_cpu_exception(void *p1, void *p2, void *p3)
 	rv = TC_FAIL;
 }
 
+void entry_cpu_exception_divide_zero(void *p1, void *p2, void *p3)
+{
+	expected_reason = K_ERR_CPU_EXCEPTION;
+
+#if defined(CONFIG_ARM64)
+	__asm__ volatile ("svc 0");
+#elif defined(CONFIG_CPU_CORTEX_R)
+	__asm__ volatile ("BKPT");
+#elif defined(CONFIG_ARM)
+	__asm__ volatile ("swi 0");
+#elif defined(CONFIG_NIOS2)
+	__asm__ volatile ("trap");
+#elif defined(CONFIG_RISCV)
+	/* use an undefined instruction to trigger
+	 * illegal instruction on RISCV
+	 */
+	__asm__ volatile (".word 0x77777777");
+#elif defined(CONFIG_ARC)
+	__asm__ volatile ("swi");
```

Comment:
This function looks like to test the scenario a divide zero exception happened, for x86, it is.  But the exceptions here for other architecture seems not intend to do this, could you please explain why you do this here? thanks!

Reply:
Ok, see the function name change, I got it.